shared/styles/style-guide-colors.js

```diff
@@ -24,9 +24,9 @@ export default {
   yellowGreen: '#a8cf36',
   yellowGreen2: '#89a82c',
 
-  lightGrey: '#f6f6f6',
-  lightGrey2: '#ebebeb',
-  lightGrey3: '#e0e0e0',
+  lightGrey: '#f0f0f0',
```

Comment:
most everything is lightGrey now. lightGrey2 is used by the buttons, grey is used by the little followed bubbles